pr/webhook/pull_request.go

```diff
@@ -117,17 +117,26 @@ func (receiver *Receiver) processPullRequest(event *github.PullRequestEvent) {
 			mentionSymbol = "@"
 		}
 		if len(handles) > 0 && !strings.Contains(*event.PullRequest.Body, "[skip notification]") {
-			body := "Notifying maintainers:\n"
+			// GitHub limits mentions to 50 per comment
+			const MentionLimit := 50
+			i := 0
+			var body string
 			for handle, ports := range handles {
+				if i % MentionLimit == 0 {
+					body = "Notifying maintainers:\n"
+				}
 				body += mentionSymbol + handle + " for port " + strings.Join(ports, ", ") + ".\n"
 				err = receiver.githubClient.AddAssignees(owner, repo, number, []string{handle})
 				if err != nil {
 					log.Println(err)
 				}
-			}
-			err = receiver.githubClient.CreateComment(owner, repo, number, &body)
-			if err != nil {
-				log.Println(err)
+				i++
```

Comment:
Please add more spacing around this (add an empty line both above and below it) and maybe add comments about how `i` works.

Reply:
I have added comments to try explaining how the counter `i` is used to split handles into batches of ≤50. But maybe what I am after is a more idiomatic way of splitting `handles` into batches of 50, and then using a nested loop over each handle in each chunk, obviating the need for a counter.